src/utils/normalization.ts

```diff
@@ -112,3 +112,31 @@ export const normalize = (payload: OperationsRepository) => {
 
   return { apis, documentation } as OperationsRepository;
 };
+
+export const emptyObject = (object: any, preserveValueKeys: string[], ignoreNestedKeys: string[]) => {
```

Comment:
This is probably too abstracted (something I sometimes also do, bad habit), but we can deal with it later on.